translations/constants.ts

```diff
@@ -0,0 +1,9 @@
+import zh_hans from './languages/zh-Hans'
+import zh_hant from './languages/zh-Hant'
+import de from './languages/de'
+
+export const translations = {
+  'zh-Hans': zh_hans,
+  'zh-Hant': zh_hant,
+  de: de
+}
```

Comment:
don't use `constants.ts`. it's an anti-pattern for scaling code, it gives a false sense of separation of concern. all it does is create a mass of code concertation that were better separated.

> An analogy for this problem is file organization in projects. Many of us have come to agree that organizing files by file type (e.g. splitting everything into html, js and css folders) don't really scale. The code related to a feature will be forced to be split between three folders, just for a false impression of "separation of concerns". The key here is that "concerns" is not defined by file type. Instead, most of us opt to organize files by feature or responsibility. https://github.com/vuejs/rfcs/issues/55#issuecomment-504875870

Reply:
@fuxingloh  The idea here is once it's added on the translations object, we automatically add it on the language checker. Otherwise, we need to re-add them on 2 files